cloudflared/Dockerfile

```diff
@@ -1,10 +1,13 @@
-ARG BUILD_FROM=ghcr.io/hassio-addons/base/amd64:12.2.7
+ARG BUILD_FROM=ghcr.io/hassio-addons/base/amd64:13.0.0
 # hadolint ignore=DL3006
 FROM ${BUILD_FROM}
 
 # Set shell
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
+# Set S6 verbosity level
+ENV S6_VERBOSITY=1
```

Comment:
Do we need this? What is the default?

Reply:
default would be "2" which outputs lines like these (info, warning & error):

```
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service init-banner: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
```

value "1" should suppress "info" messages and only logs "warning" and "error" messages. 
I guess this will reduce logs provided for issues/bugs at least a bit.